src/main/java/io/harness/cf/client/api/analytics/AnalyticsManager.java

```diff
@@ -66,15 +69,19 @@ public void pushToQueue(Target target, FeatureConfig featureConfig, Variation va
             .variation(variation)
             .eventType(EventType.METRICS)
             .build();
-
-    long sequence = ringBuffer.next(); // Grab the next sequence
+    long sequence = -1;
     try {
+      sequence = ringBuffer.tryNext(); // Grab the next sequence
       Analytics event = ringBuffer.get(sequence); // Get the entry in the Disruptor for the sequence
       event.setFeatureConfig(analytics.getFeatureConfig());
       event.setTarget(analytics.getTarget());
       event.setVariation(analytics.getVariation());
+    } catch (InsufficientCapacityException e) {
+      log.debug("Insufficient capacity in the analytics ringBuffer", e);
```

Comment:
I wonder if this should be a warn level?

Reply:
ok, I ll remove the stacktrace then